packages/svelte/tests/migrate/samples/svelte-self-skip-filename/output.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+	/** @type {Record<string, any>} */
+	let { ...props } = $props();
+</script>
+
+{#if false}
+	<!-- @migration-task: migrate this by hand or call the migrate function with the filename of this file -->
+	<svelte:self />
```

Comment:
I think it's fine to just have this as a single test, no need to check all variants for a successful addition of a comment